superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -741,28 +729,27 @@ function FiltersConfigModal({
       }
     >
       <ErrorBoundary>
-        <StyledModalBody expanded={expanded}>
-          <StyledForm
-            form={form}
-            onValuesChange={handleValuesChange}
-            layout="vertical"
+        <StyledForm
+          form={form}
+          onValuesChange={handleValuesChange}
+          layout="vertical"
+          className="test2"
```

Comment:
🙈

Reply:
hah, mean to to delete those test classes (just trying to find crap amongst all the onion layers)